neuro_san/service/watcher/main_loop/storage_watcher.py

```diff
@@ -65,24 +66,12 @@ def _run(self):
         if self.update_period_seconds <= 0:
             # We should not run at all.
             return
-        while self.go_run:
+
+        while self.keep_running:
             self.server_status.updater.set_status(True)
             time.sleep(self.update_period_seconds)
-            # Check events that may have been triggered in target registry:
-            modified, added, deleted = self.observer.reset_event_counters()
-            if modified == added == deleted == 0:
-                # Nothing happened - go on observing
-                continue
-            # Some events were triggered - reload manifest file
-            self.logger.info("Observed events: modified %d, added %d, deleted %d",
-                             modified, added, deleted)
-            self.logger.info("Updating manifest file: %s", self.manifest_path)
-
-            agent_networks: Dict[str, AgentNetwork] = \
-                RegistryManifestRestorer().restore(self.manifest_path)
-
-            public_storage: AgentNetworkStorage = self.network_storage_dict.get("public")
-            public_storage.setup_agent_networks(agent_networks)
+            for storage_updater in self.storage_updaters:
+                storage_updater.update_storage()
```

Comment:
Update each of the StorageUpdaters abstractly.  This will make it easier to add more.